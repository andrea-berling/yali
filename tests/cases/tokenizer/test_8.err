[line 2] Error: Unexpected character: %
