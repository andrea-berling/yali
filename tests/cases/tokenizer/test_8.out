LEFT_PAREN ( null
LEFT_BRACE { null
DOT . null
RIGHT_BRACE } null
RIGHT_PAREN ) null
EOF  null
